### ISSUE TITLE
Pluck is returning the id

### ISSUE THREAD

**Issue by nathany** [Casecommons/pg_search#84]
If I do a search with `pluck(:name)` it returns the `id` instead of the name (the SELECT statement isn't modified to only return the plucked field). My search scope is defined as such:

```ruby
pg_search_scope :prefix_search,
    against: :name,
    using: {tsearch: {dictionary: "english", prefix: true}}
```

**Comment by giedriusr** [comment]
@nathany is it still valid? please close the issue if not. thanks

**Comment by nathany** [comment]
I don't know. I'm not using pg_search in any projects. Feel free to close.

**Comment by nertzy** [comment]
I vote to keep this open until we have test coverage of a pluck call that works. It's most certainly due to the fact that the pg_search_rank portion of the SELECT clause is being stripped out by the call to pluck.

**Comment by giedriusr** [comment]
@nertzy sure, just trying to clean up issue tracker a bit as it is filled up with really old issues.

**Comment by asiniy** [comment]
+1 same issue

**Comment by nertzy** [comment]
Please try pluck against the current master code, I've updated how the search conditions are generated and pushed them into a subquery which should help #pluck to work properly.

**Comment by asiniy** [comment]
@nertzy, it seems all working. But when will you make a version bump?

**Comment by nertzy** [comment]
Soon! Thanks for the feedback because I want to make sure this new way of doing things actually fixes people's problems before I make such a big change.

**Comment by nertzy** [comment]
Fixed in v1.0.0
